src/common/components/repository-input/index.js

```diff
@@ -0,0 +1,70 @@
+import 'isomorphic-fetch';
+
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  changeRepositoryInput,
+  verifyGitHubRepository
+} from '../../actions/repository-input';
+
+export class RepositoryInput extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getStatusMessage() {
```

Comment:
I'm bit inconsistent here.

At first I added `statusMessage` in reducer and `updateStatusMessage` action to keep the validation/status message in the store too. But once I got to that it seemed more natural to construct status message in the component based on the state from the store.

Not sure which way to follow? Keep messages in the component (view) and remove it from reducer/action? Or move messages to the store (so the actions themselves will update the messages in the store and component will just display it?